docs/backend-system/building-plugins-and-modules/02-testing.md

```diff
@@ -6,21 +6,162 @@ sidebar_label: Testing
 description: Learn how to test your backend plugins and modules
 ---
 
-Utilities for testing backend plugins and modules are available in `@backstage/backend-test-utils`.
-`startTestBackend` returns a server which can be used together with `supertest` to test the plugins.
+Utilities for testing backend plugins and modules are available in
+`@backstage/backend-test-utils`. This section describes those facilities.
+
+## Testing Backend Plugins
+
+To facilitate testing of backend plugins, the `@backstage/backend-test-utils`
+package provides a `startTestBackend` function which starts up an entire backend
+harness, complete with a number of mock services. You can then provide overrides
+for services whose behavior you need to adjust for the test run.
```

Comment:
I think it could be good to introduce and explain the `features` option here as well, essentially the test subject(s)

Reply:
Agreed, clarified that!